Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -79,15 +80,50 @@ protected override void OnApplyTemplate()
             _listView = GetTemplateChild("ListView") as ListViewBase;
             if (_listView != null)
             {
+                this.SelectedItems = new List<object>();
```

Comment:
Wouldn't it be better to just set up a binding to the internal _listview from the control template?